code-challenges.js

```diff
@@ -23,12 +41,25 @@ var middleLetters1 = "hello"
 var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
 
-
-
+const middle = str => {
+    let newStr = ""
+    if(!(str.length %2)){
+        newStr = str.slice(str.length/2 -1,str.length/2 +1)
+    } else if(str.length %2){
```

Comment:
I would consider adding a little more logic to make your code more robust by putting `str.length % 2 === 1` instead of relying on truthy values. This code still works great, good job!